nilearn/glm/_base.py

```diff
@@ -67,6 +67,7 @@ def generate_report(
         height_control="fpr",
         min_distance=8.0,
         plot_type="slice",
+        cut_coords=None,
```

Comment:
I think that this parameter add been missing since forever eventhough the underlying make_glm_report called by this method accepts it: so that's a bug fix for me.

should I add this to the changelog?

Reply:
I think it would be good to mention in the changelog

Reply:
missed this comment:
will update the changelog post-hoc